app/bundles/PluginBundle/Controller/PluginController.php

```diff
@@ -590,11 +530,9 @@ public function reloadAction()
                 '%updated%'  => $updated,
             ]
         );
-
         $viewParameters = [
             'page' => $this->get('session')->get('mautic.plugin.page'),
         ];
-
```

Comment:
Most line spaces that were removed in this file are correctly in place per our code style. Please update this PR to reflect that. Blank lines before and after `if` blocks as well as before and after variable assignment, unless you're doing variable assignment in adjoining lines, in which case there should be no space between them. If these were removed with the post-commit `php-cs-fixer` hook, then we need to look into why, as it should not be removing them.